packages/generate-cnpj/src/cnpj.js

```diff
@@ -0,0 +1,26 @@
+export const CNPJ_LENGTH = 14;
```

Comment:
What do you think to move these constants to a constants file (like others packages)?

Reply:
Changed in bf9ab06391de583bb9a764bc6da961cbcea6f899